packages/contracts/contracts/SmartEscrowMono.sol

```diff
@@ -59,8 +59,12 @@ contract SmartEscrowMono is Context, IArbitrable, ReentrancyGuard {
     address indexed provider,
     uint256 index
   );
-  event ConfirmLocker(uint256 indexed index, uint256 indexed sum);
-  event Release(uint256 indexed index, uint256 indexed milestone);
+  event ConfirmLocker(uint256 indexed index);
+  event Release(
+    uint256 indexed index,
+    uint256 indexed milestone,
+    uint256 amount
```

Comment:
> line 63: is there a reason amount isn't included in the Release event? Is that already retrievable from somewhere else?

Good catch! I've added the amount here for easier visiblity.
But It is certainly retrievable. If we have the index we can get all the data regarding the locker from the public `lockers` variable,